framework/src/main/java/org/checkerframework/common/initializedfields/qual/EnsuresInitializedFields.java

```diff
@@ -0,0 +1,33 @@
+package org.checkerframework.common.initializedfields.qual;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.PostconditionAnnotation;
+import org.checkerframework.framework.qual.QualifierArgument;
+
+/**
+ * A method postcondition annotation indicates which fields the method definitely initializes.
+ *
+ * @checker_framework.manual #initialized-fields-checker Initialized Fields Checker
+ */
+@PostconditionAnnotation(qualifier = InitializedFields.class)
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD})
+public @interface EnsuresInitializedFields {
+    /**
+     * The object(s) whose fields this method initializes.
+     *
+     * @return object(s) whose fields are initialized
+     */
+    public String[] value() default {"this"};
```

Comment:
What does `@EnsuresInitializedFields(value = {"this", "a"}, fields = { "c", "d"})` mean?  I think that `value` should just be a string.  Then `@EnsuresInitializedFields` should be repeatable.